zcutil/build.sh

```diff
@@ -1,17 +1,17 @@
-#!/usr/bin/env bash
+#!/bin/sh
 
-set -eu -o pipefail
+set -eu
 
-function cmd_pref() {
-    if type -p "$2" > /dev/null; then
+cmd_pref() {
+    if [ type -p "$2" > /dev/null]; then
```

Comment:
The intent of the ``type -p "$2" > /dev/null`` here is to test whether "$2" exists as a command.

A POSIX ([2008 and later](https://stackoverflow.com/a/34572831/393146))-compliant way of doing this is:
```
if command -v "$2" >/dev/null; then
```

But rather than the unused generality of ``cmd_pref`` and ``gprefix``, I think it may be clearer to write:
```
if command -v greadlink >/dev/null; then
    READLINK=greadlink
else
    READLINK=readlink
fi
```

I don't know whether there's any real reason to use ``greadlink`` over ``readlink`` anyway.

Reply:
OK, it's because OS X ``readlink`` does not support ``-f``. So it is needed.